tests/filter_plugins

```diff
@@ -0,0 +1 @@
+../provision/filter_plugins
```

Comment:
Could we not add this to the sys.path in the test setUp method? I'm not clear the value of this symlink.